### ISSUE TITLE
align="end" giving inconsistent alignment for Select widget relative to other widgets when placed in the same Row in Panel 1.1.0

### ISSUE THREAD

**Issue by owenlamont** [holoviz/panel#5151]
#### ALL software version info

**In PyScript**
Browser: Vivaldi 6.0.2979.22 (Stable channel) (64-bit)
Bokeh 3.1.1
Panel 1.1.0
Pyodide 0.23.3
Pyscript 2023.03.01

**In Jupyter**
OS: Windows 11 Pro 10.0.22621
Browser: Vivaldi 6.0.2979.22 (Stable channel) (64-bit)
Packages (from conda-forge)
Bokeh 3.1.1
jupyterlab 4.0.2
Panel 1.1.0
Python 3.11.4

#### Description of expected behavior and the observed behavior

I created a Panel row as part of my application - stretched to the container width with all contained widgets aligned to the end of the Row. In Panel 0.14.4 the Select eidget along with the other widgets worked as expected and they are visually nicely aligned. In Panel 1.1.0 though the Select widget is aligned at a different vertical height to the other widgets. See attached screenshots of old and new behavior.

I've reproduced this issue both in PyScript and regular Jupyter/local installs of Panel 1.1.0

#### Complete, minimal, self-contained example code that reproduces the issue

**In PyScript**

**index.html**
```html
<!DOCTYPE html>
<html lang="en">
<head>
    <title>Panel 1.1.0 Layout Tests</title>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width,initial-scale=1">
    <link rel="stylesheet" href="https://pyscript.net/releases/2023.03.1/pyscript.css" />
    <script
            type="text/javascript"
            src="https://cdn.bokeh.org/bokeh/release/bokeh-3.1.1.js"
    ></script>
    <script
            type="text/javascript"
            src="https://cdn.bokeh.org/bokeh/release/bokeh-widgets-3.1.1.min.js"
    ></script>
    <script
            type="text/javascript"
            src="https://cdn.bokeh.org/bokeh/release/bokeh-tables-3.1.1.min.js"
    ></script>
    <script
            type="text/javascript"
            src="https://cdn.jsdelivr.net/npm/@holoviz/panel@1.1.0/dist/panel.min.js"
    ></script>
    <script defer src="https://pyscript.net/releases/2023.03.1/pyscript.js"></script>
</head>
<body>
    <py-config src="./pyscript.toml"></py-config>
    <py-script src="./main.py"></py-script>
    <div id="plot"></div>
</body>
</html>
```

**main.py**
```python
import panel as pn

button = pn.widgets.Button(name='Test', height=100)
slider = pn.widgets.IntSlider(align='end')

select_height_change = pn.widgets.Select(
    name="Height Change Id",
    width=300,
    align="end",
    disabled=True,
)
height_change_datetime_picker = pn.widgets.DatetimePicker(
    name="Height Change Time (UTC)", width=160, align="end", disabled=True
)
height_change_float_input = pn.widgets.FloatInput(
    name="Height Change (m)", width=140, align="end", disabled=True
)
confirmed_checkbox = pn.widgets.Checkbox(
    name="Confirmed", width=80, align="end", disabled=True
)
insert_button = pn.widgets.Button(
    name="Insert", width=80, align="end", disabled=True
)
delete_button = pn.widgets.Button(
    name="Delete", width=80, align="end", disabled=True
)
delete_unconfirmed_button = pn.widgets.Button(
    name="Delete Unconfirmed", width=140, align="end", disabled=True
)

pn.Row(
    select_height_change,
    height_change_datetime_picker,
    height_change_float_input,
    confirmed_checkbox,
    insert_button,
    delete_button,
    delete_unconfirmed_button,
    align="center",
    sizing_mode="stretch_width",
    width_policy="max",
).servable(target='plot')
```

**pyscript.toml**
```toml
packages = [
    "https://cdn.holoviz.org/panel/1.1.0/dist/wheels/bokeh-3.1.1-py3-none-any.whl",
    "https://cdn.holoviz.org/panel/1.1.0/dist/wheels/panel-1.1.0-py3-none-any.whl",
]

name = "Panel 1_1_0 Layout Tests"

[[interpreters]]
src = "https://cdn.jsdelivr.net/pyodide/v0.23.2/full/pyodide.js"
name = "pyodide-0.23.3"
lang = "python"

```

**In Jupyter**
```python
import panel as pn

pn.extension()

button = pn.widgets.Button(name='Test', height=100)
slider = pn.widgets.IntSlider(align='end')

select_height_change = pn.widgets.Select(
    name="Height Change Id",
    width=300,
    align="end",
    disabled=True,
)
height_change_datetime_picker = pn.widgets.DatetimePicker(
    name="Height Change Time (UTC)", width=160, align="end", disabled=True
)
height_change_float_input = pn.widgets.FloatInput(
    name="Height Change (m)", width=140, align="end", disabled=True
)
confirmed_checkbox = pn.widgets.Checkbox(
    name="Confirmed", width=80, align="end", disabled=True
)
insert_button = pn.widgets.Button(
    name="Insert", width=80, align="end", disabled=True
)
delete_button = pn.widgets.Button(
    name="Delete", width=80, align="end", disabled=True
)
delete_unconfirmed_button = pn.widgets.Button(
    name="Delete Unconfirmed", width=140, align="end", disabled=True
)

pn.Row(
    select_height_change,
    height_change_datetime_picker,
    height_change_float_input,
    confirmed_checkbox,
    insert_button,
    delete_button,
    delete_unconfirmed_button,
    align="center",
    sizing_mode="stretch_width",
    width_policy="max",
)
```

#### Screenshots or screencasts of the bug in action

**Old 0.14.4 Behavior in PyScript**
![image](https://github.com/holoviz/panel/assets/12672027/70fa6d41-2f87-4921-8bf5-fe513c1508f2)

**New 1.1.0 Behavior in PyScript**
![image](https://github.com/holoviz/panel/assets/12672027/a496209b-575b-4537-9b33-ee7be45b42ab)

**New 1.1.0 Behavior in Jupyter**
![image](https://github.com/holoviz/panel/assets/12672027/b1b80185-a992-4c07-9df8-a15d48282232)

**Comment by philippjfr** [comment]
Should be fixed in 1.1.1

**Comment by owenlamont** [comment]
Cool, I can confirm it's good now
